starboard/contrib/RDK/README.md

```diff
@@ -0,0 +1,17 @@
+### RDK Cobalt Open Source reference port
+
+RDK Starboard reference port for the RDK-V profile is maintained by the RDK team.
+It is currently targeted to build on Amlogic AH212 reference device.
+
+Starboard implementation can be found at
+https://code.rdkcentral.com/r/plugins/gitiles/rdk/components/generic/cobalt
+
+For a more detailed overview of RDK reference port, please see the following:
+
+Youtube Partner portal:
+
+https://support.google.com/yt-lr-device/answer/12174345?hl=en&ref_topic=12174437
```

Comment:
This link should go to https://developers.google.com/youtube/devices/living-room/compliance/ah212 now that the partner portal has migrated to DevSite.

Reply:
Thanks updated, exactly the kind of feedback i was hoping for ! That's a lot nicer link too.

Reply:
This is a good point. https://developers.google.com/youtube/devices/living-room/access/accessing-lr-partnership-resources, along with the homepage, is accessible to the public. You could add a line stating something like, If you don't have access, see Accessing Living Room Partnership Resources to set it up.

Reply:
Added the link link - have another look but i think it's fine to go in for now.